README.md

```diff
@@ -0,0 +1,235 @@
+# Question-answering-with-extracted-text-summarization
+This project is to answer questions related to keywords using google with extracted text summarization.
+
+# Functional Design
```

Comment:
For you functional design, please use `snake_case` for your python functions (you are currently using `camelCase`)

Reply:
I have changed the camelCase to snake_case. Thanks.